tests/algorithms.test.ts

```diff
@@ -1,6 +1,113 @@
-import { MADe, M_Estimator, Q, Hampel, A_Algorithm } from '../src/lib';
+import {
+  MADe,
+  M_Estimator,
+  Q,
+  Hampel,
+  A_Algorithm,
+  ReferenceValue,
+} from '../src/lib';
 
 describe('Algorithms', () => {
+  it('Reference Value (CASE1: x value is not range)', () => {
+    let xSample = 0.4;
```

Comment:
We can drop `sample`. Since it's a test file, it's clear these are sample values.

So just `x` and `formulae` should be enough.